src/components/ErrorComponent.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const ErrorComponent = () => {
+  return (
+    <div>ErrorComponent</div>
```

Comment:
in this ErrorComponent. you can import useRouteError hook from react-router-dom. and assign to a variable and display  some StatusText or error code. 

I had done like this:
import { useRouteError } from "react-router-dom";

const ErrorComponent = () => {
  const errorData = useRouteError();
  return (
    <>
      <h1>Error Component</h1>
      <h2> Error : {errorData.statusText}</h2>
    </>
  );
};

export default ErrorComponent;

Reply:
great!